openmc/mgxs/library.py

```diff
@@ -350,7 +350,7 @@ def build_library(self):
     def add_to_tallies_file(self, tallies_file, merge=True):
         """Add all tallies from all MGXS objects to a tallies file.
 
-        NOTE: This assumes that build_library() has been called
+        NOTE: This assumes that :meth:`Library.build_library` has been called
```

Comment:
It is actually [not a typo](http://www.sphinx-doc.org/en/stable/domains.html#role-py:meth).